v1/models/work_spaces.py

```diff
@@ -14,24 +14,14 @@ class WorkSpace(Base):
     deleted_at = Column(DateTime)
     user_id = Column(Integer, ForeignKey("users.id"), nullable = False)
 
-    @classmethod
-    def __camel_to_snake(cls, input_str: str) -> str:
-        output_str = "".join(['_' + i.lower() if i.isupper() else i for i in input_str]).lstrip('_')
-        return output_str
-
     @classmethod
     def __snake_to_camel(cls, input_str: str) -> str:
```

Comment:
hàm snake to camle dùng trong các Model được viết lặp đi lặp rất nhiều lần, cần đưa vào class chung hoặc utils